app/facades/search_facade.rb

```diff
@@ -34,4 +34,12 @@ def self.find_reviews(id)
       CreateReview.new(review)
     end
   end
+
+  def self.find_recommendations(id, api_key)
+    result = MovieInfo.recommendations(id, api_key)
+    recommended = result[:results].map do |movie|
+      CreateRecommendation.new(movie)
+    end
+    recommended[0..4]
```

Comment:
Might have to adjust some minor things with the new refactor, but should be quick